client/src/controllers/ZoneController.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class ZoneController extends Controller<HTMLElement> {
+    static classes = ['active'];
+
+    declare activeClasses: string[];
```

Comment:
```suggestion
    /** ... some description */
    declare activeClasses: string[];
```

While it's quite obvious at a glance, it's best to include a basic description about this.